addons/mail/static/src/components/message_in_reply_to_view/message_in_reply_to_view.xml

```diff
@@ -8,7 +8,8 @@
                     <span class="o_MessageInReplyToView_wrapOuter d-flex align-items-center text-muted opacity-75 opacity-100-hover cursor-pointer"  t-attf-class="{{ messageInReplyToView.messageView.isInChatWindowAndIsAlignedRight ? 'pe-3': 'ps-3' }}" t-on-click="messageInReplyToView.onClickReply">
                         <img class="o_MessageInReplyToView_avatar me-2 rounded-circle" t-att-src="messageInReplyToView.messageView.message.parentMessage.avatarUrl" t-att-title="messageInReplyToView.messageView.message.parentMessage.authorName" alt="Avatar"/>
                         <span class="o_MessageInReplyToView_wrapInner overflow-hidden">
-                            <b class="o_MessageInReplyToView_author">@<t t-out="messageInReplyToView.messageView.message.parentMessage.authorName"/></b>:
+                            <b class="o_MessageInReplyToView_author">@<t t-out="messageInReplyToView.messageView.message.parentMessage.authorName"/></b>
+                            <br t-if="messageInReplyToView.messageView.isInChatWindow"/>
                             <span class="o_MessageInReplyToView_body ms-1 text-break">
```

Comment:
Could you limit the bread to attachment link? It doesn't feel necessary when this is not, and showing break all the time is not pretty, especially when the message is aligned right (the text is aligned on the left)

Reply:
> Could you limit the bread to attachment link? It doesn't feel necessary when this is not, and showing break all the time is not pretty, especially when the message is aligned right (the text is aligned on the left)

I did not get your concern here. Will you please flesh it out in detail? :)

Reply:
hey @alexkuhn, 

That means you want the `<br>` tag only for the left side. After your suggestion is 
implemented it will look like this Is it ok for you? 

![Screenshot from 2023-09-27 12-56-32](https://github.com/odoo/odoo/assets/80447598/f7a0f20b-c8ad-4f7b-9e49-a2d264c3b857)

Reply:
Yes, thank you :)